_posts/2022-04-24-sr2022-knockouts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: TEAM 1st wins Student Robotics Competition 2022!
+image: image
+image_alt: Alt text for the image
+description: SR2022 came to an exciting conclusion with a victory for HRS
+---
+
```

Comment:
Suggestion: Might be nice to add the group photo to the page too like https://studentrobotics.org/blog/2019-04-09-sr2019-tlc-crowned-champion/

Link: https://github.com/srobo/team-emails/pull/110#discussion_r859031450